gnucash/import-export/aqb/gnc-plugin-aqbanking.ui

```diff
@@ -62,7 +62,7 @@
     <item>
       <attribute name="label" translatable="yes">_Online Banking Setup…</attribute>
       <attribute name="action">gnc-plugin-aqbanking-actions.ABSetupAction</attribute>
-      <attribute name="tooltip" translatable="yes">Initial setup of Online Banking access (HBCI, or OFX DirectConnect, using AqBanking)</attribute>
+      <attribute name="tooltip" translatable="yes">Initial setup of Online Banking access (FinTS or OFX DirectConnect) using AqBanking</attribute>
```

Comment:
Shouldn't this be "FinTS, OFX DirectConnect, or Swiss YellowNet"?

Reply:
Google just reminded me of https://lists.gnucash.org/pipermail/gnucash-user/2021-November/098561.html
The yellownet module was dropped. Only it's file importer still exists—see File->Import->Import using AqBanking.
That means we should remove any mention of yellownet in the setup process, …